dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -857,6 +860,7 @@
       "description": "Every relation is an entity graph of a certain relationType, where all entities are connected and have specific roles within the relation context.",
       "required": [
         "relationType",
+        "confidenceScore",
```

Comment:
should we make it optional? if a relation does not have a confidence score we shoud not have it in the output, right?

Reply:
Correct. If it won't always be in the response, it should not be required.

Reply:
Removed confidenceScore from required